src/BuiltinHelper.js

```diff
@@ -152,9 +152,14 @@ class BuiltinHelper extends Helper {
      * Fetch an asset but don't process dependencies.
      * @param {AssetType} assetType - The type of asset to fetch.
      * @param {string} assetId - The ID of the asset to fetch: a project ID, MD5, etc.
-     * @return {Promise.<Asset>} A promise for the contents of the asset.
+     * @return {!Promise.<Asset>} A promise for the contents of the asset.
```

Comment:
The `!` means "never null" -- did you mean to add `?`?
(I think `?` is optional for reference types -- implied unless `!` is present -- but I'm a little fuzzy on that.)
See also: http://usejsdoc.org/tags-type.html

```suggestion
     * @return {?Promise.<Asset>} A promise for the contents of the asset.
```